src/common/copilot/IntelligenceApiService.ts

```diff
@@ -15,12 +15,13 @@ import { EXTENSION_NAME } from "../../client/constants";
 const clientType = EXTENSION_NAME + '-' + getExtensionType();
 const clientVersion = getExtensionVersion();
 
-export async function sendApiRequest(userPrompt: UserPrompt[], activeFileParams: IActiveFileParams, orgID: string, apiToken: string, sessionID: string, entityName: string, entityColumns: string[], telemetry: ITelemetry, aibEndpoint: string | null, geoName: string | null) {
+export async function sendApiRequest(userPrompt: UserPrompt[], activeFileParams: IActiveFileParams, orgID: string, apiToken: string, sessionID: string, entityName: string, entityColumns: string [], telemetry: ITelemetry, aibEndpoint: string | null, geoName: string | null) {
```

Comment:
NIT: Can you remove this added whitespace in order to consistency throughout the code.